Description: link with gtop-2.0
 The gtop library reads information about processs and the state of the system.
Forwarded: no
Author: Andres Salomon <dilinger@voxel.net>
Last-Update: 2011-03-22
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/lib/Apache2/Build.pm
+++ b/lib/Apache2/Build.pm
@@ -449,7 +449,7 @@
         $xlibs .= ' -lintl';
     }
 
-    return " -lgtop -lgtop_sysdeps -lgtop_common $xlibs";
+    return " -lgtop-2.0 -lgtop_sysdeps-2.0 -lgtop_common-2.0 $xlibs";
 }
 
 sub gtop_ldopts {
